packages/origin-device-registry-irec-local-api/src/utils/success-response.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ISuccessResponse } from '@energyweb/origin-backend-core';
+import { IsBoolean, IsString, ValidateIf } from 'class-validator';
+
+export class SuccessResponseDTO implements ISuccessResponse {
```

Comment:
This is used across our system. Maybe we can add this definition to `@energyweb/origin-backend-utils`.